lib/discord_bot/bot.rb

```diff
@@ -114,7 +114,7 @@ def initialize
       ]
     end
 
-    def run(skip_motd: false, fast_boot: false)
+    def run(skip_motd: false, fast_boot: true)
```

Comment:
Try not to commit this change, I need to make it a flag you can pass dynamically during testing

Reply:
How would you recomend I resolve this change? it should be set false right now, you want it to be set as true?